### ISSUE TITLE
Valid keys slipping through as invalid and not showing up missing

### ISSUE THREAD

**Issue by jon2992** [glebm/i18n-tasks#207]
We had an issue where we had an invalid key used in our views, but we didn't notice until it went to production because it raises a TranslationMissingError.

```yaml
# Translations
en:
  parent:
    original_value: 'Some text'
```

```ruby
# Usage in HAML

%p= t('parent.original_value:')
```

In this instance, with `strict: true`, the key is seen as [invalid on this line](https://github.com/glebm/i18n-tasks/blob/v0.9.5/lib/i18n/tasks/scanners/pattern_scanner.rb#L34).  Because of that, it's never processed. I'm not sure how you would want to handle this. Maybe providing a list of invalid keys and their usage in the application? Letting them go through and be used as missing translation?

As far as I can tell though, I18n allows this kind of key in my testing...

```yaml
# sample other translation
en:
  parent:
    'some_key:': 'A super special value'
```

```ruby
# This evalutes
%p= t('parent.some_key:')
```

**Comment by jon2992** [comment]
Conceivably a fix for this would be modifying [this line](https://github.com/glebm/i18n-tasks/blob/master/lib/i18n/tasks/scanners/ruby_key_literals.rb#L22) to include a `:` as part of the `VALID_KEY_CHARS`.

**Comment by glebm** [comment]
Including `:` in `VALID_KEY_CHARS` sounds good to me

**Comment by jon2992** [comment]
Welp. Should've realized it would pick up on those commits, my bad. Got a pull request open, though. :)

**Comment by glebm** [comment]
Released in v0.9.6!
